inc/init.php

```diff
@@ -350,23 +354,43 @@ function init_files(){
  *
  * @return bool|string
  */
-function init_path($path){
+function init_path($path,$required=null){
```

Comment:
Could you please add comment before the function about the behavior of `$required`? I think this should be a boolean, not based on NULL or !NULL.

Reply:
Both good ideas. Thank you @phy25.